### DISCUSSION TITLE
Whisper server on docker

### DISCUSSION THREAD

**Discussion by lvca** [ggml-org/whisper.cpp#2233] [Q&A]
I see there is support for a server (https://github.com/ggerganov/whisper.cpp/tree/master/examples/server) with this distribution. How can I start the server in a docker image so I can access whisper via HTTP?

**Comment by albertusgeyser** [comment]
Looks like we looking for same answer as i also want a way to run whisper cpp in docker.
I tried this:
https://github.com/sumeetdas/whisper.cpp-docker
But not working, and i guess if you waiting so long for help i would also wait for a long time?
